hoard/source.py

```diff
@@ -12,7 +15,12 @@ def __iter__(self) -> Iterator[Dataset]:
         return self
 
     def __next__(self) -> Dataset:
-        ...
+        record = next(self.client)
+        xml = BeautifulSoup(record, "html.parser")
+        json_url = xml.metadata["directapicall"]
+        dataverse_json = requests.get(json_url).json()
```

Comment:
We may want to consider creating a `requests.Session` object in `__init__()` and using that to make requests, just to cut down on the number of connections we're making. It's already a lot.

Reply:
Yeah, good call. I'll go ahead and do that.